_episodes/05-function-style.md

```diff
@@ -72,34 +89,48 @@ canvas(with_attribution=True)
 ~~~
 {: .output}
 
-If we try the same thing on our `mean` function, we don't get a helpful error message.
+If we try the same thing on our `open_pdb` function, we don't get a helpful error message.
```

Comment:
I know 'error' was present before, but that is a mistake. Need to remove the word error here, as it is not an error message.